pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -38,16 +38,25 @@ spec:
             - "--extra-create-metadata=true"
             - "--prevent-volume-mode-conversion=true"
             - "--feature-gates=HonorPVReclaimPolicy=true"
-            {{- if .EnableCSITopology }}
+            {{ if .CSILogRotation }}
+            - "--logtostderr=false"
+            - "--alsologtostderr=true"
+            - "--log_file={{ .CsiLogDirPath }}/cephcsi/log/rbd-provisioner/csi-provisioner.log"
+            {{ end }}
```

Comment:
isnt it we already decided we will log only from cephcsi/csiaddons containerers not from sidecar containers?

Reply:
So we will creating log file for 

> Ceph-CSI provisioners
> Ceph-CSI plugins/drivers
> CSI Addons 

So deployment.yam will log for,

1) csi-provisioner
2) csi-addons
3) csi-rbdplugin

And rbd daemon plugin.yaml will log for,

1) csi-rbdplugin
2) csi-addons

I hope all of these are ceph-csi containers?

Reply:
> So we will creating log file for
> 
> > Ceph-CSI provisioners
> > Ceph-CSI plugins/drivers
> > CSI Addons
> 
> So deployment.yam will log for,
> 
> 1. csi-provisioner

This is a sidecar container which we don't own/ship, its a kubernetes-csi project

> 2. csi-addons
> 3. csi-rbdplugin
> 
> And rbd daemon plugin.yaml will log for,
> 
> 1. csi-rbdplugin
> 2. csi-addons
> 
> I hope all of these are ceph-csi containers?

Reply:
Okay thanks for the clarification, updating it